src/3-Prayer-Request/PrayerRequestDisplay.tsx

```diff
@@ -184,11 +186,10 @@ const PrayerRequestDisplay = ({navigation, route}:PrayerRequestDisplayProps):JSX
                                     setPrayerRequestEditModalVisible(false);
                                     if (prayerRequestData !== undefined && prayerRequestListData !== undefined) setPrayerRequestState(prayerRequestData, prayerRequestListData);
                                     else if (deletePrayerRequest == true) {
-                                        // remove prayer request from list page
-                                        route.params.callback !== undefined && route.params.callback();
-                                        navigation.pop();
+                                        dispatch(removeOwnedPrayerRequest(appPrayerRequestListItem.prayerRequestID));
+                                        dispatch(removeNewPrayerRequest(appPrayerRequestListItem.prayerRequestID)); // remove from dashboard
+                                        navigation.goBack();
```

Comment:
Dashboard `newPrayerRequestList` are not owned to be deleted.  

If another user in a separate instance deletes their prayer request, then:
1. `CircleDisplay` wouldn't return
2. `PrayerRequestList` is fetching the latest
3. If deletion occurs after fetching the list, it would just a 404 failed API call attempting to open `PrayerRequestDisplay`

I think this level of syncing we can address later

Reply:
Per our conversation on discord, if a user shares a prayer request with a circle they are a member of, they become both the owner and recipient of said prayer request. Thus, those prayer requests CAN show up on the dashboard. That is what this new reducer aims to address.

Unfortunately the same will happen on the circles page, which is not handled by redux.

Reply:
Sorry to be stuck on this; dashboard `newPrayerRequestList` should really be read only.
It's the same query as recipient list in `PrayerRequestList`, but limited to 7 items.  (So not complete).

I do see the potential issue, where they share with a circle; they are also a member of that circle, as such are a recipient of their own.  
=> I'll add a condition to the query to prevent an owned prayer request showing in either of these lists.

*Circles are filtered via `circleID`; the potential for overlap there still exists.  If it's easier, we could make view only if accessed from circle page?